lib/src/base-styles/_nx-btn.scss

```diff
@@ -21,7 +21,7 @@
   @include container-horizontal;
   @extend %nx-btn--disabled;
 
-  background-color: $nx-white;
+  background-color: var(--rsc-component-background-color);
```

Comment:
I don't think that we want to use this variable on buttons, we'd want a separate set for the various button types

Reply:
> I don't think that we want to use this variable on buttons, we'd want a separate set for the various button types

aa490bf3